### DISCUSSION TITLE
Alt+Enter doesn't work if chat box is focused

### DISCUSSION THREAD

**Discussion by plaguemask** [ppy/osu#16950] [Q&A] [answered]
Alt+Enter is treated as just pressing Enter in the chat box. I've ran into this problem when looking at a playlist as the chat box is always focused, but it could also occur if you have the chat overlay open. It's mildly inconvenient to have to back out of a screen/overlay to switch window modes in this manner.

**Comment by peppy** [comment, marked as answer]
Will be resolved via https://github.com/ppy/osu-framework/pull/5048.
